### PR TITLE
Fix msys2-mingw build on AppVeyor

### PR DESCRIPTION
**Pull request by ja2-derek**
AppVeyor build broken since #1118, because the script `.github/scripts/ci-buidl.sh` was split. I didn't realize it has been used by AppVeyor msys2-mingw build.

### CODE REVIEW

**Comment by ja2-derek** [comment]
Fixed. We should be disabling that msys2 build though.

File: .github/scripts/ci-build.sh [hunk 1]
```diff
@@ -46,15 +46,14 @@ if [[ "$CI_TARGET" == "linux" ]]; then
...
-  export RUSTUP_INIT_ARGS="${RUSTUP_INIT_ARGS} --target=x86ST_64-pc-windows-gnu --profile=minimal"
   export RUN_TESTS=false
 
 elif [[ "$CI_TARGET" == "msys2-mingw32" ]]; then
   export CMAKE_GENERATOR="MSYS Makefiles"
   export CONFIGURE_CMD="${CONFIGURE_CMD} -DCPACK_GENERATOR=ZIP"
   export RUSTUP_HOME="$(cygpath -w ~/.rustup)"
   export CARGO_HOME="$(cygpath -w ~/.cargo)"
-  export RUSTUP_INIT_ARGS="${RUSTUP_INIT_ARGS}-i686-pc-windows-gnu --default-host=i686-pc-windows-gnu"
+  export PATH=$PATH:$HOME/.cargo/bin
```
Comment (lynxlynxlynx) [inline .github/scripts/ci-build.sh:56]:
```suggestion
  export PATH="$PATH:$HOME/.cargo/bin"
```
In case the paths ever get whitespace in them

**Comment by lynxlynxlynx** [comment]
Why the change from the minimal profile? Not good enough or did those two extra components make it identical to the default. Or something third?

**Comment by ja2-derek** [comment]
> The default profile includes all of components in the minimal profile, and adds rust-docs, rustfmt, and clippy.

Using `minimal` we need to install fmt and clippy with a second command. I do not see any significant difference in the time spent between default and minimal + fmt + clippy.

**Pull request merged**